### ISSUE TITLE
get_default_config() gives "Error in readRDS(file) : unknown input format"

### ISSUE THREAD

**Issue by dweemx** [dynverse/babelwhale#21]
Hi,

Error is raised when I'm trying to `infer_trajectory`:
![image](https://user-images.githubusercontent.com/16015936/66046064-44a93000-e525-11e9-96e8-252a3656cfdc.png)

I do set the configuration using:
```
library(babelwhale)
config<-babelwhale::create_singularity_config(
  cache_dir = "./singularity-images/"
)
babelwhale::set_default_config(config = config)
```

Any idea what could cause the problem?

Cheers,
M

**Comment by rcannood** [comment]
Hey Maxime, 

Maybe something went wrong while saving the config file to disk. Also, it is better to use an absolute path for your cache dir. Could you try saving the config again and reading the config from disk?

```r
config <- babelwhale::create_singularity_config("absolute_path_to_cache_dir")
babelwhale::set_default_config(config)
babelwhale::get_default_config()
```

Robrecht

**Comment by dweemx** [comment]
Hi Robrecht, 
I already put an absolute path 
![image](https://user-images.githubusercontent.com/16015936/66118985-fe60d900-e5d7-11e9-96d3-94de3ad89981.png)
(Btw, a few months ago it did work)

**Comment by rcannood** [comment]
Aha, seems I made a stupid mistake. Give me a few minutes to fix it.

**Comment by dweemx** [comment]
Works now thanks !

**Comment by rcannood** [comment]
FYI, the bug fix has been pushed to CRAN in version 1.0.1 of babelwhale. Thanks for discovering the bug! :)
